### DISCUSSION TITLE
YamlCreate.ps1 Submit PR issue

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [microsoft/winget-pkgs#97274] [General] [answered]
Why submit PR after sandbox test is defaulting to submitting to PR `default { $PromptSubmit = '0' }` ?. This is invoking PR submit on any key press other than `N`

**Comment by Trenly** [comment, marked as answer]
> Why submit PR after sandbox test is defaulting to submitting to PR `default { $PromptSubmit = '0' }` ?. This is invoking PR submit on any key press other than `N`

It defaults to that because that is what most users choose to do. You can change it to always require manual submission in the [YamlCreate Settings](https://github.com/microsoft/winget-pkgs/blob/master/doc/tools/YamlCreate.md#yamlcreate-settings) by setting `AutoSubmitPRs: false`

You can also remove the default action by setting `ExplicitMenuOptions: true`
